src/common/IOSUtils.ts

```diff
@@ -170,13 +170,9 @@ export class IOSUtils {
             }
         );
 
-        if (rtIntersection.length > 0) {
-            return new Promise<string[]>((resolve, reject) =>
-                resolve(rtIntersection)
-            );
-        } else {
-            return new Promise<string[]>((resolve, reject) => reject());
-        }
+        return new Promise<string[]>((resolve, reject) =>
```

Comment:
This didn't match the consumption pattern of this method. Namely, a lack of responsive runtimes shouldn't be an error at this level. It should just return the empty list of runtimes.